root/static/scripts/tests/typeInfo.js

```diff
@@ -4785,7 +4785,7 @@ MB.relationshipEditor.exportTypeInfo({
          "cardinality1" : 0,
          "deprecated" : false,
          "id" : 281,
-         "orderableDirection" : 0,
+         "orderableDirection" : 1,
```

Comment:
That just specifies which entity is being ordered. It can be set to either 1=forward (entity1 is orderable) or 2=backward (entity0 is orderable). It only applies to link types in the `orderable_link_type` table, which is where `orderableDirection` comes from.

In this case it's 1, so entity1 is orderable:
WorkA has part WorkB (link_order: 1)
WorkA has part WorkC (link_order: 2)
WorkA has part WorkD (link_order: 3)

Some link types have an `orderableDirection` of 2, e.g. the event<->series part-of link type:
EventA (link_order: 1) is part of SeriesA
EventB (link_order: 2) is part of SeriesA
EventC (link_order: 3) is part of SeriesA

The entities in an l_ table are positioned alphabetically, e.g. l_**E**vent_**S**eries, so in some cases it's just determined by that.